gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Put <code>CREATE EXTENSION</code> in code font</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>